src/pages/addCard/components/addCard/AddCardForm.tsx

```diff
@@ -0,0 +1,63 @@
+import { useAddCardForm } from '@pages/addCard/hooks';
+import {
+	CardNumberInput,
+	CardExpiredDateInput,
+	CardHolderNameInput,
+	CardSecurityCodeInput,
+	CardPasswordInput,
+} from '@pages/addCard/components';
+import { Button } from '@components/ui-kit';
+import { PaymentCard } from '@components/domain';
+
+export default function AddCardForm() {
+	const {
+		displayedCardNumber,
+		displayedExpiredDate,
+		cardHolderName,
+		cardSecurityCode,
+		firstCardPassword,
+		secondCardPassword,
+		handleCardNumberChange,
+		handleCardExpiredDateChange,
+		handleCardHolderNameChange,
+		handleCardSecurityCodeChange,
+		handleFirstCardPasswordChange,
+		handleSecondCardPasswordChange,
+		handleSubmit,
+	} = useAddCardForm();
+
+	return (
+		<form onSubmit={handleSubmit}>
+			<PaymentCard
+				cardNumber={displayedCardNumber}
+				cardExpiredDate={displayedExpiredDate}
+				cardHolderName={cardHolderName}
+			/>
+			<CardNumberInput
+				value={displayedCardNumber}
+				onChange={handleCardNumberChange}
+			/>
+			<CardExpiredDateInput
+				value={displayedExpiredDate}
+				onChange={handleCardExpiredDateChange}
+			/>
+			<CardHolderNameInput
+				value={cardHolderName}
+				onChange={handleCardHolderNameChange}
+			/>
+			<CardSecurityCodeInput
+				value={cardSecurityCode}
+				onChange={handleCardSecurityCodeChange}
+			/>
+			<CardPasswordInput
+				firstValue={firstCardPassword}
+				secondValue={secondCardPassword}
+				onFirstValueChange={handleFirstCardPasswordChange}
+				onSecondValueChange={handleSecondCardPasswordChange}
+			/>
+			<div style={{ width: '100%', textAlign: 'right' }}>
```

Comment:
```suggestion
			<div style={{ width: '100%', textAlign: 'right' }}>
```
이 부분은 항상 리렌더링이 될 것 같습니다.
이유가 뭘까요!?